fn/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName= 'functions'
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://github.com/iron-io/functions/releases/download/0.2.25/fn.exe'
```

Comment:
Unfortunately this URL is bound to that version. We might need to adjust scripts to auto increase the version. Updated version to 0.2.36.